extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -88,3 +88,46 @@ export class WizardController {
 		});
 	}
 }
+
+export function createInformationRow(view: azdata.ModelView, label: string, value: string): azdata.FlexContainer {
+	return view.modelBuilder.flexContainer()
+		.withLayout(
+			{
+				flexFlow: 'row',
+				alignItems: 'center',
+			})
+		.withItems(
+			[
+				creaetLabelTextComponent(view, label),
+				createTextCompononent(view, value)
+			],
+			{
+				CSSStyles: { 'margin-right': '5px' }
+			})
+		.component();
+}
+
+export function createHeadingTextComponent(view: azdata.ModelView, value: string): azdata.TextComponent {
+	const component = createTextCompononent(view, value);
+	component.updateCssStyles({
+		'font-size': '13px',
+		'font-weight': 'bold'
+	});
+	return component;
+}
+
+
+export function creaetLabelTextComponent(view: azdata.ModelView, value: string): azdata.TextComponent {
+	const component = createTextCompononent(view, value);
+	component.updateCssStyles({
+		'color': '#595959',
```

Comment:
Validated that this meets accessibility requirements for dark/light/HC themes?

Reply:
removing this for now